src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -14,124 +14,74 @@
 
 namespace Microsoft.DotNet.Tools.Publish
 {
-    public class Program
+    public class PublishCommand
     {
-        public static int Main(string[] args)
-        {
-            DebugHelper.HandleDebugSwitch(ref args);
+        public string ProjectPath;
+        public string Configuration;
+        public string OutputPath;
+        public string Framework;
+        public string Runtime;
+        public NuGetFramework NugetFramework;
+        public IEnumerable<ProjectContext> ProjectRidFxPermutations;
```

Comment:
I thought we switched to nuget so that we can start making slightly breaking changes - especially when it doesn't break the code and only binary compat. I will switch it but I think the only think what is changing is few more characters in the code.